login.css

```diff
@@ -88,12 +88,12 @@ ul li ul{
 }
 /* Set up the footer */
 .footer{
-	background-color: #C8A2C8;
+	background-color: #0C59CF;
```

Comment:
06/08/2021: I change the footer background color to navy blue since I it can display a good contrast can help user to aim at important location can also I tried to make the color theme be consistency and don't apply too many color.